spec/facility_spec.rb

```diff
@@ -1,26 +1,92 @@
+=======
+require 'rspec'
+require './lib/facility'
+require './lib/vehicle'
+
+>>>>>>> feature/vehicle-class
 RSpec.describe Facility do
   before(:each) do
-    @facility = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+<<<<<<< HEAD
+<<<<<<< HEAD
+    # Initialize a facility with a hash of details
+    @facility = Facility.new({ 
+      name: 'DMV Main Branch', 
+      address: '123 Main St', 
+      phone: '555-1234' 
+=======
+    @facility = Facility.new({
+      name: 'DMV Main Branch',
+      address: '123 Main St',
+      phone: '555-1234'
+    })
+
+    @vehicle = Vehicle.new({
+      vin: '123456789ABCDEFG',
+      year: 2000,
+      make: 'Toyota',
+      model: 'Camry',
+      engine: :ice
+>>>>>>> feature/vehicle-class
+    })
+  end
+
+  it 'initializes with details and no services' do
+    expect(@facility.name).to eq('DMV Main Branch')
+    expect(@facility.address).to eq('123 Main St')
+    expect(@facility.phone).to eq('555-1234')
+    expect(@facility.services).to eq([])
+    expect(@facility.registered_vehicles).to eq([])
+    expect(@facility.collected_fees).to eq(0)
+  end
+
+  it 'can add services' do
+    @facility.add_service('Vehicle Registration')
+    expect(@facility.services).to include('Vehicle Registration')
+<<<<<<< HEAD
+    expect(@facility.services).to include('Renew License')
+    expect(@facility.services.length).to eq(2) # Check the count of services
+=======
+    # Update to pass a hash for initialization
+    @facility = Facility.new({
+      name: 'DMV Main Branch',
+      address: '123 Main St',
+      phone: '555-1234'
+    })
   end
-  describe '#initialize' do
-    it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('DMV Tremont Branch')
-      expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
-    end
+
+  it 'can initialize with attributes' do
+    expect(@facility.name).to eq('DMV Main Branch')
+    expect(@facility.address).to eq('123 Main St')
+    expect(@facility.phone).to eq('555-1234')
+    expect(@facility.services).to eq([])
+  end
+
+  it 'can add services' do
+    @facility.add_service('Vehicle Registration')
+    @facility.add_service('Road Test')
+
+    expect(@facility.services).to include('Vehicle Registration', 'Road Test')
+>>>>>>> feature/facility-class
+=======
   end
 
-  describe '#add service' do
-    it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
-    end
+  it 'can register a vehicle and collect fees' do
```

Comment:
It seems like this test was added but is not implemented. Be sure to run all your tests to make sure things are green! It's also helpful to refer back to the interaction pattern in order to prevent missing functionality